js/index.js

```diff
@@ -1,7 +1,53 @@
-// Iteration 1: Names and Input
+// Iteration 1
+const hacker1 = 'Carlos'
+const hacker2 = 'Esteban'
 
+console.log(`The driver name is ${hacker1}`)
+console.log(`The navigator name is ${hacker2}`)
 
-// Iteration 2: Conditionals
+// Iteration 2
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length}`)
+} else if (hacker2.length > hacker1.length) {
+    console.log(`It seems the navigator has the longest name, it has ${hacker2.length}`)
+} else if (hacker2.length === hacker1.length) {
+    console.log(`Wow, you have both have equally long names, ${hacker2.length}`)
+}
 
+// Iteration 3
+console.log(hacker1.split('').join(' ').toUpperCase())
+console.log(hacker1.split('').reverse().join(''))
```

Comment:
esto esta bien por que no especificaba que tenian que hacer pero la mejor practica es hacer una funcion y consologearla.

`let str2 = ""
for(i = hacker2.length-1; i >= 0; i--){
  str2 += `${hacker2[i]}`
}
console.log(str2)`

En este caso creamos una variable vacia, recorremos el nombre en direccion inversa y vamos agregandolo a la variable nueva.